### DISCUSSION TITLE
Other Language Support

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#571] [Ideas]
I saw in another beta project for a game a way for the community to participate in translating the game to another language, is it possible to do with harpy?
And I think people are better than google especially when translating.

**Comment by robertodoering** [comment]
Yeah, community driven translation for harpy would be great! Plenty of people have already offered their support in translating harpy into their native language through email, which is pretty awesome.
Currently harpy doesn't support translations, but support for it will be implemented in the near future.

My plan is to then host a translation tool online which volunteers can use to add new translations (as opposed to having to create a PR on github).

**Reply by FerasK0** [reply]
Very cool!
I'll be waiting for it, a lot of people i know are holding off because of the language so yeah.
